Kubernetes Invoker tests deploy to OpenShift cluster if logged in

The Kubernetes Invoker tests deploy to an OpenShift cluster if you're logged in to one and fail:
```
[ERROR] The following builds failed:
[ERROR] *  kubernetes-docker-build-and-deploy/pom.xml
```
We need to fix that and log out from the cluster before running these tests.

/cc @geoand @iocanel

--- a/integration-tests/kubernetes/maven-invoker-way/src/it/kubernetes-docker-build-and-deploy/invoker.properties
+++ b/integration-tests/kubernetes/maven-invoker-way/src/it/kubernetes-docker-build-and-deploy/invoker.properties
@@ -1,4 +1,5 @@
 # invoker.goals=clean package -Dquarkus.container.build=true -Dquarkus.package.type=native 
-invoker.goals=clean package -Dquarkus.kubernetes.deploy=true
+# ensure that an attempt to deploy is made, but that the attempt fails (as we don't want to deploy this test application to a cluster that the runner of test may have configured)
+invoker.goals=clean package -Dquarkus.kubernetes.deploy=true -Dquarkus.kubernetes-client.master-url=http://localhost:12345
 # expect a failure since there is no Kubernetes cluster to deploy to
 invoker.buildResult = failure

--- a/integration-tests/kubernetes/maven-invoker-way/src/it/kubernetes-jib-build-and-deploy/invoker.properties
+++ b/integration-tests/kubernetes/maven-invoker-way/src/it/kubernetes-jib-build-and-deploy/invoker.properties
@@ -1,4 +1,5 @@
-# invoker.goals=clean package -Dquarkus.container.build=true -Dquarkus.package.type=native 
-invoker.goals=clean package -Dquarkus.kubernetes.deploy=true
+# invoker.goals=clean package -Dquarkus.container.build=true -Dquarkus.package.type=native
+# ensure that an attempt to deploy is made, but that the attempt fails (as we don't want to deploy this test application to a cluster that the runner of test may have configured)
+invoker.goals=clean package -Dquarkus.kubernetes.deploy=true -Dquarkus.kubernetes-client.master-url=http://localhost:12345
 # expect a failure since there is no Kubernetes cluster to deploy to
 invoker.buildResult = failure

--- a/integration-tests/kubernetes/maven-invoker-way/src/it/openshift-docker-build-and-deploy/invoker.properties
+++ b/integration-tests/kubernetes/maven-invoker-way/src/it/openshift-docker-build-and-deploy/invoker.properties
@@ -1,4 +1,5 @@
-# invoker.goals=clean package -Dquarkus.container.build=true -Dquarkus.package.type=native 
-invoker.goals=clean package -Dquarkus.kubernetes.deploy=true
+# invoker.goals=clean package -Dquarkus.container.build=true -Dquarkus.package.type=native
+# ensure that an attempt to deploy is made, but that the attempt fails (as we don't want to deploy this test application to a cluster that the runner of test may have configured)
+invoker.goals=clean package -Dquarkus.kubernetes.deploy=true -Dquarkus.kubernetes-client.master-url=http://localhost:12345
 # expect a failure since there is no Kubernetes cluster to deploy to
 invoker.buildResult = failure

--- a/integration-tests/kubernetes/maven-invoker-way/src/it/openshift-s2i-build-and-deploy/invoker.properties
+++ b/integration-tests/kubernetes/maven-invoker-way/src/it/openshift-s2i-build-and-deploy/invoker.properties
@@ -1,4 +1,5 @@
-# invoker.goals=clean package -Dquarkus.container.build=true -Dquarkus.package.type=native 
-invoker.goals=clean package -Dquarkus.kubernetes.deploy=true
+# invoker.goals=clean package -Dquarkus.container.build=true -Dquarkus.package.type=native
+# ensure that an attempt to deploy is made, but that the attempt fails (as we don't want to deploy this test application to a cluster that the runner of test may have configured)
+invoker.goals=clean package -Dquarkus.kubernetes.deploy=true -Dquarkus.kubernetes-client.master-url=http://localhost:12345
 # expect a failure since there is no Kubernetes cluster to deploy to
 invoker.buildResult = failure

